modules/@apostrophecms/asset/lib/webpack/apos/webpack.scss.js

```diff
@@ -45,6 +45,9 @@ module.exports = (options, apos) => {
             {
               loader: 'sass-loader',
               options: {
+                sassOptions: {
+                  silenceDeprecations: [ 'mixed-decls' ]
```

Comment:
do we need this one? As sass mentioned in their documentation:

> 
> ⚠️ Heads up!
>
> The deprecated functionality you’re depending on will eventually break.

Reply:
Val requested that we add this until we go through and fix all the other modules (at least the ones that we load in testbed).

Reply:
I don't think this is a good idea.
We need to know where are the deprecated features.

Reply:
I'm ok!

Reply:
I'm adding the suggested changes then